code/modules/mod/mod_theme.dm

```diff
@@ -195,6 +195,29 @@
 				SEALED_CLOTHING = STOPSPRESSUREDAMAGE,
 			),
 		),
+		"titan" = list(
+			HELMET_LAYER = NECK_LAYER,
+			HELMET_FLAGS = list(
+				UNSEALED_CLOTHING = SNUG_FIT,
+				SEALED_CLOTHING = THICKMATERIAL|STOPSPRESSUREDAMAGE,
+				UNSEALED_INVISIBILITY = HIDEFACIALHAIR,
+				SEALED_INVISIBILITY = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDESNOUT,
+				SEALED_COVER = HEADCOVERSMOUTH|HEADCOVERSEYES|PEPPERPROOF,
```

Comment:
this part, the helmet seems to be a helmet that fully extends over the head even when unsealed, so the flags/layer should reflect that